### ISSUE TITLE
Update all existing components to use the new focus style

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#395]
```[tasklist]
### Tasks
- [ ] https://github.com/digdir/designsystem/issues/362
- [ ] https://github.com/digdir/designsystem/issues/426
- [x] https://github.com/digdir/designsystem/issues/613
- [ ] https://github.com/digdir/designsystem/issues/421
- [x] Be om inspill på Slack
- [ ] https://github.com/digdir/designsystem/issues/833
```

**Comment by mrosvik** [comment]
Blocked by https://github.com/digdir/designsystem/issues/833

**Comment by mrosvik** [comment]
Møte om 9.01.24 om sak https://github.com/digdir/designsystemet/issues/833

Beslutninger:

Textfield, textarea, combobox, search beholder gul focus style ved klikk
Radio, checkbox, switch, tabs, card blir som de er (gul ved tastaturnavigering, ikke ved klikk)
Link endres til å kun vise gulfarge ved tastaturnavigering (ikke ved klikk) PR https://github.com/digdir/designsystemet/pull/1403
Button endres til å ha gul bakgrunn ved tastaturnavigering (i stedet for gul outline). Skal ikke bli gul ved klikk. https://github.com/digdir/designsystemet/issues/820
Vi skriver en artikkel om begrunnelsen for valgene som er tatt. Begynner diskusjon og argumentasjon her i https://github.com/digdir/designsystemet/issues/997

**Comment by mrosvik** [comment]
Replaced by what is decided in #1409
